### ISSUE TITLE
cpu 1700%

### ISSUE THREAD

**Issue by kangkang59812** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#59]
I run the code on linux server using gpu. But cpu occupies too high.Is there anything wrong?

**Comment by sgrvinod** [comment]
Hi @kangkang59812 , it doesn't happen for me. Does this happen when you run `train.py`?

**Comment by kangkang59812** [comment]
@sgrvinod yes. And my linux server has 20 core cpu. It seems like every core is 100%. Only reading images and pre-processing use cpu , right?

**Comment by sgrvinod** [comment]
Yes, pretty much. Also, the dataloader uses just the main thread on the CPU I think. Does the same happen if you don't do the training but just the validation?
